README.md

```diff
@@ -1,3 +1,247 @@
-# swift-quickstart-platform
+# PubNub Swift Sample App
 
 This repository contains the files required to run the [PubNub Swift Quickstart](https://www.pubnub.com/docs/platform/quickstarts/swift).
+
+PubNub takes care of the infrastructure and APIs needed for the realtime communication layer of your application. Work on your app's logic and let PubNub handle sending and receiving data across the world in less than 100ms.
+
+## Get keys
+
+You will need the publish and subscribe keys to authenticate your app. Get your keys from the [Admin Portal](https://dashboard.pubnub.com/login).
+
+## Setup the project
+
+If you don't want to copy the code from this document, you can clone the repository and use the files in there.
+
```

Comment:
'and use the files in there', can we say something better here? like 'clone the repository and start the xcode project' or something similar.

Reply:
Suggestion above.